cli/local-net

```diff
@@ -0,0 +1,198 @@
+            f'{taraxa_cli} account --secret {private_key} | grep address:', shell=True).strip().decode('utf-8').split(' ')[1]
+        private_keys[worker_name] = private_key[2:]
+        public_addresses[worker_name] = public_address
+
+        boot_nodes_flags.append(f'--boot-node 0.0.0.0:10{i+1}02/{public_key[2:]}')
+
+    boot_nodes_flags = " ".join(boot_nodes_flags)
+
+    base_path = os.path.realpath(os.path.dirname(
+        os.path.realpath(__file__)) + '/../')
+    template = json.loads('{}')
+    with open(f'{base_path}/config/networks/{network}.json') as json_file:
+        template_data = json.load(json_file)
+        template.update(template_data)
+
+    eligibility_balance_threshold = template['chain_config'][
+        'final_chain']['state']['dpos']['eligibility_balance_threshold']
+    genesis_state = template['chain_config']['final_chain']['state']['dpos']['genesis_state']
+
+    for i in genesis_state:
+        genesis_state[i] = {}
+        for a in public_addresses:
+            genesis_state[i][public_addresses[a]] = eligibility_balance_threshold
+
+    chainId = hex(randint(500, 1000))
+    timestamp = hex(int(time.time()))
+
+    for i in range(boot_nodes):
+        worker_name = f'boot-node-{i}'
+        config_name = f'./local-net/conf-{worker_name}.json'
+
+        if not os.path.isdir(f'{local_net}/db-{worker_name}'):
+            os.mkdir(f'{local_net}/db-{worker_name}')
+
+        cmd = ''
+        cmd += f'{taraxa_cli} config node --network {network} '
+        cmd += f'{boot_nodes_flags} '
+        cmd += '--as-boot-node '
+        cmd += f'--node-secret {private_keys[worker_name]} '
+        cmd += f'--prop-override network_tcp_port:10{i+1}02 --prop-override network_udp_port:10{i+1}02 '
+        cmd += f'--prop-override rpc.http_port:70{i+1}7 --prop-override rpc.ws_port:80{i+1}7 '
+        cmd += f'--prop-override db_path:{local_net}/db-{worker_name} '
+        cmd += f'--prop-override chain_config.chain_id:{chainId} '
+        cmd += f'--prop-override chain_config.dag_genesis_block.timestamp:{timestamp} '
+        cmd += f'--prop-override chain_config.final_chain.genesis_block_fields.timestamp:{timestamp} '
+        cmd += f'--file {config_name}'
+
+        os.system(cmd)
+
+        config = json.loads('{}')
+        with open(config_name) as json_file:
+            config_data = json.load(json_file)
+            config.update(config_data)
+
+        config['chain_config']['final_chain']['state']['dpos']['genesis_state'] = genesis_state
+
+        with open(config_name, 'w') as json_file:
+            json.dump(config, json_file, indent=2)
+
+    for i in range(rpc_nodes):
+        worker_name = f'rpc-node-{i}'
+        config_name = f'./local-net/conf-{worker_name}.json'
+
+        if not os.path.isdir(f'{local_net}/db-{worker_name}'):
+            os.mkdir(f'{local_net}/db-{worker_name}')
+
+        cmd = ''
+        cmd += f'{taraxa_cli} config node --network {network} '
+        cmd += f'{boot_nodes_flags} '
```

Comment:
I think this won't work since the nodes need to connect to the boot nodes, and the address here are for listening.
You may change everything to `127.0.0.1` on line 45 that it may work.

But I am not sure.

Reply:
A side note, when i tried to run with `0.0.0.0` it kept showing this:
` 2021-05-19 18:36:28.945035 SUMMARY SILENT STATUS: NOT CONNECTED TO ANY PEERS. POSSIBLE CONFIG ISSUE OR NETWORK CONNECTIVITY`
when I changed to `127.0.0.1`  that msg did not show up.

Reply:
Changed to 127.0.0.1 and it works. Fixed